### PR TITLE
base-settings: add button functionality

### PR DESCRIPTION
**Pull request by Tomasvrba**
This PR implements functionality for these settings buttons:
- Enable/Disable Tor
- Change hostname
- Change password
- Shut down
- Create backup
- Enable ssh password authentication
- Set system password

Additionally:
- Add number of active lightning channels

### CODE REVIEW

**Comment by Tomasvrba** [comment]
@TheCharlatan here is the PR for the Base which introduced the ssh password changes: https://github.com/digitalbitbox/bitbox-base/pull/237

File: backend/bitboxbase/bitboxbase.go [hunk 1]
```diff
@@ -599,6 +599,11 @@ func (base *BitBoxBase) SetLoginPassword(password string) error {
 	if !reply.Success {
 		return &reply
 	}
+	// FIXME: Change backend node management to status: offline
+	err = base.Deregister()
```
Comment (TheCharlatan) [inline backend/bitboxbase/bitboxbase.go]: Why are you deregistering here?

Comment (Tomasvrba) [reply backend/bitboxbase/bitboxbase.go]: Yep, good catch, that shouldn't be there :) :100:

Comment (TheCharlatan) [reply backend/bitboxbase/bitboxbase.go]: We should have just checked the code :P

Comment (TheCharlatan) [reply backend/bitboxbase/bitboxbase.go]: What is the FIXME for?

Comment (Tomasvrba) [reply backend/bitboxbase/bitboxbase.go]: That one also belongs to the `RebootBase()` method. 
I will remove it altogether here. 
Part of what I'm working on for the backend base management, auto-reconnects and different statuses based on what kind of restart/disconnect/shutdown happens.
This was a reminder to change the `Deregister()` to something else when restarting.

**Pull request merged**